
graspit_msgs/Grasp grasp
---
float64 success_probability
